{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "a8ea836a-cfe6-44e4-9d70-19e938519691-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "inData", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1604655189591, "stop": 1604655189619, "uuid": "b3cac188-f9a1-441e-8fd0-e4ee302f1683", "historyId": "7eb54adce62304271e7e440620b1c341", "testCaseId": "f56f10c6b6679a14e1ccd1e8f399d334", "fullName": "test_userManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_userManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "13656-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_userManage"}]}